tests/tracer/test_trace_utils.py

```diff
@@ -160,3 +162,42 @@ def test_set_http_meta_custom_errors(mock_log, span, int_config, error_codes, st
         mock_log.exception.assert_called_once_with(*log_call)
     else:
         mock_log.exception.assert_not_called()
+
+
+def test_activate_distributed_headers_enabled(int_config):
+    pin = Pin(tracer=Tracer())
```

Comment:
```suggestion
```

don't actually need a pin in these tests anymore 🙂

Reply:
Thanks for the catch 🤒